contrib/kafka/config/README.md

```diff
@@ -21,7 +21,7 @@ Deployment steps:
    `my-cluster-kafka-bootstrap.mynamespace:9092`.
 1. Apply the 'Kafka' ClusterChannelProvisioner, Controller, and Dispatcher:
    ```
-   ko apply -f config/provisioners/kafka/kafka.yaml
```

Comment:
Could we (not this PR) but get the release done for this so it could be applied with kubectl as well?